sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -440,6 +440,7 @@ def post_port_sfp_info_to_db(logical_port_name, port_mapping, table, transceiver
             break
 
         if not _wrapper_get_presence(physical_port):
+            helper_logger.log_warning("Transceiver not present in port {}".format(logical_port_name))
```

Comment:
@mihirpat1 may be notice should be sufficient.

Reply:
@prgeor I have now modified it to notice.